legacy-sdk/whirlpool/tests/integration/multi-ix/adaptive_fee.test.ts

```diff
@@ -338,8 +350,8 @@ describe("adaptive fee tests", () => {
     console.info(oppositeTx?.meta?.logMessages);
   });
 */
-  describe("trade with AdaptiveFee", () => {
-    describe("swap", () => {
+  describe("trade with AdaptiveFee (LiteSVM)", () => {
```

Comment:
curious, what does the appended `(LiteSVM)` indicate? is it differentiating from some other set of tests?